instance/test_config.py.default

```diff
@@ -11,3 +11,6 @@ USER_MAX_SERVERS = 10
 USER_MAX_TEAMS = 100
 USER_MAX_MATCHES = 1000
 DEFAULT_PAGE = '/matches'
+
+
+SPECTATOR_IDS = []
```

Comment:
I'd rather this not be added here, unless you're actually testing this feature.

Reply:
I will be testing this feature this weekend extensively. I have taken the modifications to my own build and will report back if it works as @tloesch describes

Reply:
I meant in the unit tests, which is the purpose of this file.

Reply:
@splewis i just added it to pass the pull request tests

Reply:
Yes. Otherwise you're breaking people that update without adding this to their config.

Reply:
@splewis 
You are right. I add the code this week.